.github/workflows/reviewdog.yml

```diff
@@ -27,7 +34,7 @@ jobs:
   full-review:
     runs-on: ubuntu-latest
     container:
-      image: golang:1.18
+      image: golang:1.20
```

Comment:
this should use the same setup-go action as the other job in this file. so, remove the "container:" section and add "install go" task - same as above